### DISCUSSION TITLE
My github page doesn't work

### DISCUSSION THREAD

**Discussion by GRUMCZAK** [community/community#147107] [Other Feature Feedback, Questions, & Ideas]
### SOLVED: Select Topic Area

Question

### Body

I've tried to set up a custom domain and I accidentally set up one I don't posess. Now I can't undo it so that my page has the default address once again. Below is the option that should reset site address to default one (empty Custom Domain field).

![image](https://github.com/user-attachments/assets/edf97fbb-0206-4411-937c-e43ef882ccab)

SOLUTION: I've deleted the repo and made another one - the default domain was used to host my github site on.

**Comment by thelaycon** [comment]
- Clone your Github page repo on your local machine.
- Delete old repo.
- Push to a new repo with a different name and initiate a Github page on that.
